client/app/components/Input/InputTag.jsx

```diff
@@ -21,12 +21,14 @@ export type State = {
   checkboxes: Checkbox[],
   autocompleteLabel?: string,
   autoHighlight: boolean,
+  suggestions: string[]
 };
 
+
 export class InputTag extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-    this.state = { checkboxes: props.checkboxes, autoHighlight: false };
+    this.state = { checkboxes: props.checkboxes, autoHighlight: false, suggestions: [], autocompleteLabel: '' };
```

Comment:
I think you can remove `autocompleteLabel` since it can either be undefined or a string as defined on line 22.